lib/mparticle/models/commerce_event.rb

```diff
@@ -51,7 +52,8 @@ def self.attribute_map
         :'shopping_cart' => :'shopping_cart',
         :'currency_code' => :'currency_code',
         :'screen_name' => :'screen_name',
-        :'is_non_interactive' => :'is_non_interactive'
+        :'is_non_interactive' => :'is_non_interactive',
+        :'custom_flags' => :'custom_flags'
```

Comment:
maybe worth adding trailing comma here if it's safe and that's what we're doing elsewhere?